tools/after-platform-add-android.js

```diff
@@ -1,31 +1,23 @@
-//--------------------------------------
-// Useful functions
-//--------------------------------------
 var fs = require('fs');
 var path = require('path');
-var targetAndroidApi = 23;
 
-// Function to fix AndroidManifest.xml
+// Function to fix AndroidManifest.xml.
 var fixAndroidManifest = function(data) {
     
-    // Fix application tag
+    // Fix application tag.
     var appName = "com.salesforce.androidsdk.phonegap.app.HybridApp";
     
-    // In case the script was run twice
+    // In case the script was run twice.
     if (data.indexOf(appName) == -1) {    
         var applicationTag = '<application android:hardwareAccelerated="true" android:icon="@drawable/sf__icon" android:label="@string/app_name" android:manageSpaceActivity="com.salesforce.androidsdk.ui.ManageSpaceActivity" android:name="' + appName + '">'
         data = data.replace(/<application [^>]*>/, applicationTag);
         
-        // Comment out first activity
+        // Comment out first activity.
         data = data.replace(/<activity/, "<!--<activity");
         data = data.replace(/<\/activity>/, "</activity>-->");
         
-        // Change min sdk version
-        data = data.replace(/android\:minSdkVersion\=\"10\"/, 'android:minSdkVersion="19"');
-        
-        // Change target api
-        data = data.replace(/android\:targetSdkVersion\=\"22\"/, 'android:targetSdkVersion="' + targetAndroidApi + '"');
-        
+        // Change min SDK version.
+        data = data.replace(/android\:minSdkVersion\=\"14\"/, 'android:minSdkVersion="19"');
```

Comment:
This is a different bugfix. I took the opportunity to cleanup some incorrect steps we had.